src/app/shared/palette/document/write-document-field.component.ts

```diff
@@ -48,7 +48,7 @@ export class WriteDocumentFieldComponent extends AbstractFieldWriteComponent imp
           document._links.binary.href,
           document.originalDocumentName,
         );
-
+        this.caseField.value = this.selectedFile.name;
```

Comment:
shouldn't we also set the caseField.value to null later on the way we set 
`      this.selectedFile = null;
`
on line 58?

Reply:
also, aren't we using this.caseField.value already with a different meaning? on the ngOnInit:
```ngOnInit() {
    let document = this.caseField.value;
    if (document) {
      this.createDocumentGroup(
        document.document_url,
        document.document_binary_url,
        document.document_filename,
      );
    }
  }